TCAT/Model/Route.swift

```diff
@@ -198,6 +198,10 @@ class Route: NSObject, JSONDecodable {
         }
     }
     
+    func isWalkingRoute() -> Bool {
+        return routeSummary.count == 2 && routeSummary.first?.nextDirection == .walk
```

Comment:
I know backend was sometimes returning odd walking directions with 3 steps, maybe to be safe, just run a .all functional check to make sure all directions are walk maybe